src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public abstract class Figure implements InformationDrawer, AreaCalculator {
+    protected Colors color;
```

Comment:
It is better to use the String as a Type
```suggestion
    protected String color;
```